xbmc/pvr/providers/PVRProviders.cpp

```diff
@@ -106,7 +106,7 @@ bool CPVRProviders::LoadFromDatabase(const std::vector<std::shared_ptr<CPVRClien
   if (database)
   {
     CPVRProviders providers;
-    database->Get(providers, clients);
+    m_iLastId = database->Get(providers, clients);
```

Comment:
Instead of removing const from CPVRDatabase::Get, make this member mutable, this is exactly what this keyword is for, to keep logical constness of a method.

Reply:
The mutability problem extends itself from the DB wrapper that is called. So setting this variable to be mutable does not help.

Reply:
Ah, now I see.